query/src/main/scala/filodb/query/exec/ExecPlan.scala

```diff
@@ -264,17 +264,22 @@ trait ExecPlan extends QueryCommand {
     */
   protected def args: String
 
+  // TODO(a_theimer): more generic way to handle lambda param with default?
```

Comment:
If you've got a more scala-thonic way to do this, let me know-- Google has not been super helpful.
I want something like:
```
ExecPlan => T = (ep) => 0
```